src/pymor/core/config.py

```diff
@@ -28,6 +29,24 @@ def _get_fenics_version():
     return df.__version__
 
 
+def _get_dunegdt_version():
+    import dune.xt
+    import dune.gdt
+    version = 'outdated'
+    try:
+        if parse(dune.gdt.__version__) < parse('2021.1.2') or version >= parse('2021.2'):
```

Comment:
Could you create a single source of truth for the current/tested version(-string)? It would lessen the chance of missing an instance to edit when updating.